### DISCUSSION TITLE
Optional peer dependencies

### DISCUSSION THREAD

**Discussion by ssvendsen** [glideapps/glide-data-grid#314] [General]
Hi. The "marked" and "react-responsive-carousel" peer dependencies are only needed (I think) if using the markdown cell type and image cell type (or its editor). Could there be a way to not have to install these peer dependencies if we're not using these cell types?

**Comment by jassmith** [comment]
Patches are welcome. I'm not really sure what happens if you don't install them.

**Comment by lukasmasuch** [comment]
It's working fine for me not having `marked` and `react-responsive-carousel` in my dependencies. However, I need it in my dev dependencies to have my tests running.

**Comment by jassmith** [comment]
I haven't tried yet but what happens if you don't install them?
